### PR TITLE
Homework to lesson 07 is here

### PR DESCRIPTION
**Pull request by Karolina777**
Hi,

Here's the homework to lesson 7
Please, review

KR,
Natalia

### CODE REVIEW

File: 07/hmw-07.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+phone_book = dict()
+
+# create the initial phone book
+while True:
```
Comment (klymenok) [inline 07/hmw-07.py]: This part is redundant since you can still work with an empty phonebook and fill it in the next loop. I would suggest removing it.

**Review by klymenok — CHANGES_REQUESTED** [review]

File: 07/hmw-07.py (hunk 1 above)
Comment (Karolina777) [reply 07/hmw-07.py]: Hi, I have commented the 1st loop which was to create the phone book

File: 07/hmw-07.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+phone_book = dict()
+
+# create the initial phone book
```
Comment (klymenok) [inline 07/hmw-07.py:3]: For the future: there is no need to keep unused code commented. You can delete it. If you need this code as an example, you can keep it in some other file and don't commit that file

**Review by klymenok — APPROVED** [review]
Looks good

**Pull request merged**